tests/integrate_test/test_integrate_scores.py

```diff
@@ -280,13 +280,13 @@ def test_service_flag(self):
 
         table = {}
         for flag in IconServiceFlag:
-            if flag.name is 'scorePackageValidator':
+            if flag.name is 'SCORE_PACKAGE_VALIDATOR':
```

Comment:
I think you meant using `SERVICE_SCORE_PACKAGE_VALIDATOR`, better use the constant variable instead of the constant value

Reply:
This FP was remained some problems.
so @soobokjin is going to make bug_fix FR